Sources/CMakeLists.txt

```
@@ -12,6 +12,36 @@
 ##
 ##===----------------------------------------------------------------------===##
 
+add_library(FoundationBuildMacros INTERFACE)
+export(TARGETS FoundationBuildMacros FILE FoundationBuildMacros.cmake)
+if(SwiftFoudation_MACRO)
```

Aha, I think this is the problem

```suggestion
if(SwiftFoundation_MACRO)
```

---

Doh! Good catch. Thanks.